### PR TITLE
Update zypper.go (opensuse-tumbleweed)

### PR DESCRIPTION
**Pull request by wgjtyu**
- Add package(webkit2gtk3-soup2-devel) for `libwebkit`.
- Add package(pkgconf-pkg-config) for `pkg-config`.
- Change environment variable before running command to query information of the package.

### CODE REVIEW

File: v2/internal/system/packagemanager/zypper.go [hunk 1]
```diff
@@ -60,7 +62,9 @@ func (z *Zypper) PackageInstalled(pkg *Package) (bool, error) {
 	if pkg.SystemPackage == false {
 		return false, nil
 	}
-	stdout, _, err := shell.RunCommand(".", "zypper", "info", pkg.Name)
+	var env []string
+	env = shell.SetEnv(env, "LANGUAGE", "en_US.utf-8")
```
Comment (leaanthony) [inline v2/internal/system/packagemanager/zypper.go:66]: Do we know this isn't going to break non US English installations?

Comment (wgjtyu) [reply v2/internal/system/packagemanager/zypper.go:66]: Yes, my OS was Chinese, if we don't set `LANGUAGE` env variable here, the `zypper info pkgName` command will return result in Chinese, so `wails doctor` will nott work properly here.

**Comment by leaanthony** [comment]
Please update the changelog in `website/src/pages/changelog.mdx` 🙏

**Comment by wgjtyu** [comment]
OK, updated.

**Review by leaanthony — APPROVED** [review]

**Review by leaanthony — APPROVED** [review]

**Pull request merged**